README.md

```diff
@@ -1,58 +1,173 @@
+
+### High-level overview
+
+|                                 | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
 |:--------------------------------|:------:|:------:|:------:|:-------:|
 | Single binary                   | ![yes] | ![yes] | ![yes] | ![no]   |
 | Language                        | Rust   | Rust   | Go     | Python  |
-| Syntax highlighting             | ![yes] | ![yes] | ![yes] | ![yes]  |
-| Sass compilation                | ![yes] | ![yes] | ![yes] | ![yes]  |
-| Assets co-location              | ![yes] | ![yes] | ![yes] | ![yes]  |
-| Multilingual site               | ![ehh] | ![no]  | ![yes] | ![yes]  |
-| Image processing                | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Sane & powerful template engine | ![yes] | ![yes] | ![ehh] | ![yes]  |
-| Themes                          | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Shortcodes                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Internal links                  | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Link checker                    | ![yes] | ![no]  | ![no]  | ![yes]  |
-| Table of contents               | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Automatic header anchors        | ![yes] | ![no]  | ![yes] | ![yes]  |
+| Syntax highlighting             | [![yes]](https://www.getzola.org/documentation/content/syntax-highlighting/) | ![yes] | ![yes] | ![yes]  |
+| Sass compilation                | [![yes]](https://www.getzola.org/documentation/content/sass/) | ![yes] | ![yes] | ![yes]  |
+| Multilingual site               | [![ehh]](https://www.getzola.org/documentation/content/multilingual/) | ![no]  | ![yes] | ![yes]  |
+| Image processing                | [![yes]](https://www.getzola.org/documentation/content/image-processing/) | ![no]  | ![yes] | ![yes]  |
+| Link checker                    | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#check) | ![no]  | ![no]  | ![yes]  |
+| Search                          | [![ehh]](https://www.getzola.org/documentation/content/search/) | ![no]  | ![no]  | ![yes]  |
+| Data files                      | [![yes]](https://www.getzola.org/documentation/templates/overview/#load-data) | ![yes] | ![yes] | ![no]   |
+| LiveReload                      | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#serve) | ![no]  | ![yes] | ![yes]  |
+
+**Notes:**
+
+- *Multilingual site*
+  - zola gets ![ehh] because despite having a basic translations system, it is being redesigned to evade its current limitations (see [discussion on the forum](https://zola.discourse.group/t/rfc-internationalization-system-rework/546))
+- *Search*
+  - zola gets ![ehh] because search engine is disabled by default in some languages, as explained [in the docs](https://www.getzola.org/documentation/content/multilingual/#configuration)
```

Comment:
I'm not sure about that being a `ehh`. There is a flag built-in if you need to enable it for Chinese/Japanese but to get a `yes` we would need to give everyone a 90MB binary where 80% is something a good chunk of users are not going to use? It doesn't sound like an improvement to me.

Reply:
I agree the tradeoff is good, but don't you think this limitation should be advertised in the comparison table?